packages/web-twig/src/Resources/components/UNSTABLE_EmptyState/UNSTABLE_EmptyState.twig

```diff
@@ -4,20 +4,18 @@
 {%- set _buttonPrimaryUrl = props.buttonPrimaryUrl | default(null) -%}
 {%- set _buttonSecondaryText = props.buttonSecondaryText | default(null) -%}
 {%- set _buttonSecondaryUrl = props.buttonSecondaryUrl | default(null) -%}
-{%- set _description = props.description -%}
+{%- set _description = props.description | default(null) -%}
+{%- set _heading = props.heading | default(null) -%}
 {%- set _headingElementType = props.headingElementType | default('h2') -%}
-{%- set _iconName = props.iconName | default(null) -%}
+{%- set _imageSlot = props.imageSlot | default(null) -%}
```

Comment:
```suggestion
{%- set _unsafeImage = props.UNSAFE_image | default(null) -%}
```

We have an agreement with our users-developers that all props that are rendered using `raw` are prefixed with the `UNSAFE_` prefix because they are not escaped by the Twig/Symfony.